security/compliance_operator/compliance-operator-tailor.adoc

```diff
@@ -0,0 +1,15 @@
+[id="compliance-operator-tailor"]
+= Tailoring the Compliance Operator
+include::modules/common-attributes.adoc[]
+:context: compliance-tailor
+
```

Comment:
s/will need to be/must be

    s/*tailoring*/_tailoring_

per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference